internal/resources-validation.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+parameters:
+  # Git ref of tk-toolchain to use.
+  tk_toolchain_ref: master
+  # Git ref of tk-core to use.
+  tk_core_ref: master
```

Comment:
I don't see those parameters being used in the following job. Can they be removed or will you need them for testing?